weekly-updates/weekly-update.2015-09-04.md

```diff
@@ -0,0 +1,64 @@
+### Node.js News — September 4th
+Node.js v4.0.0 is released
+
+### Node.js v4.0.0
+
+This week is the most historical week. Finally we have released Node.js v4.0.0(stable). All thanks to the every Node/io.js collaborator. This Node.js is the most stable Node ever. We have reviewed all of patches and the state of the test suite is more stable and there are more active contributors in new Node.
```

Comment:
`All thanks to the every Node/io.js collaborator.` -> `Thanks to every Node/io.js collaborator!`